lightning/src/ln/channel.rs

```diff
@@ -378,6 +379,11 @@ pub const FEE_SPIKE_BUFFER_FEE_INCREASE_MULTIPLE: u64 = 2;
 #[cfg(not(fuzzing))]
 const FEE_SPIKE_BUFFER_FEE_INCREASE_MULTIPLE: u64 = 2;
 
+/// If we fail to see a funding transaction confirmed on-chain within this many blocks after the
+/// channel creation on an inbound channel, we simply force-close and move on.
```

Comment:
A couple of questions:

- this is just for inbound, right?  for outbound, I think you must double spend a funding input to safely consider the channel closed
- what does "force-close" mean exactly here?  I normally think of force-close as going onchain, but here we want to just forget the channel, right?

Reply:
> this is just for inbound, right? for outbound, I think you must double spend a funding input to safely consider the channel closed

Correct, yes.

> what does "force-close" mean exactly here? I normally think of force-close as going onchain, but here we want to just forget the channel, right?

Correct, yes. Because we don't have any funds in the channel we don't care what happens, we just send the peer an error message and let them deal with it. (we may, in fact, try to broadcast our latest closing state, but it presumably wont work given the funding transaction isn't confirmed).

Reply:
I see you answered the first question in this PR, but maybe it also makes sense to update this comment clarifying the above two points.

Reply:
Hmm, the comment there does say "on an inbound channel", do you have a suggestion for a better way to phrase it? As for "force-close", I mean I believe we *do* do the same thing as always here - we send our peer an error message, mark the channel as gone in `ChannelManager`, and broadcast our latest local state. Of course our latest local stay may not propagate, but funding may, in fact, be in the mempool.

Reply:
OK, agreed on all points, and this is the right behavior in a future with anchors / dual funding / push, so we can try to claim our funds when possible.